### PR TITLE
Updated to provide instructions for multiple Pis

### PR DESCRIPTION
**Pull request by jimbobbennett**
## Description

Added what to do if you already have a Pi using the default hostname on your network.

## Type of PR
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Checklist
- [ ] ~~Code follows this projects style guide~~
- [X] I reviewed my code
- [ ] ~~Tested in deployment environment~~
- [ ] ~~Tested in an alternate environment (please update here: )~~
- [X] Required documentation has been updated
- [ ] ~~Other (explain: )~~

### CODE REVIEW

File: headless-setup/README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ In the following section you'll connect to Raspberry Pi from you computer, chang
 1. Connect your Raspberry Pi to a power source
 1. On your computer open Command Prompt (windows) or Terminal (mac)
 1. Type `ssh pi@raspberrypi.local -o UserKnownHostsFile=/dev/null`. *Note: we use the **UserKnownHostsFile=/dev/null** option here in order to not save the hostname to our ssh Known Host record. We only do this the first time we connect because we're going to change the Host Name.*
+    > If you already have a Pi using the hostname `raspberrypi.local` you will need to connect using the Pi's IP address. Use the command `ssh pi@<IP_Address>`, where `<IP_Address>` is the IP address, for example `192.168.0.2`. You can find the IP address either using your routers management software, or using the [Adafruit Pi Finder utility](https://learn.adafruit.com/the-adafruit-raspberry-pi-finder/finding-and-connecting).
```
Comment (adiazulay) [inline headless-setup/README.md]:
```suggestion
    > If you already have a Pi using the hostname `raspberrypi.local` you will need to connect using the Pi's IP address. Use the command `ssh pi@<IP_Address>`, where `<IP_Address>` is the IP address, for example `192.168.0.2`. You can find the IP address either using your routers management software, or by disconnecting the other Raspberry Pi from the network and running `ping raspberrypi.local -4` in command prompt. 
```

**Review by adiazulay — CHANGES_REQUESTED** [review]
The Adafruit Utility has some known issues with Pi 4s and finding multiple Pis on the same network. I added instructions on how to find the IP in command prompt.

File: headless-setup/README.md (hunk 1 above)
Comment (jimbobbennett) [reply headless-setup/README.md]: Is this a good solution? If you are using the other Pi for a particular task would you want to have to unplug it? Any reason for not using the Adafruit Pi finder?

Comment (adiazulay) [reply headless-setup/README.md]: The adafruit finder [doesn't detect Pi 4](https://github.com/adafruit/Adafruit-Pi-Finder/issues/78)s, and can't [detect multiple Pis](https://github.com/adafruit/Adafruit-Pi-Finder/issues/68) connected to the same network. 

I would say this solution is just ok, it'll get you unstuck if you you already have `raspberrypi.local` saved in .ssh folder.

Comment (jimbobbennett) [reply headless-setup/README.md]: I learned something today! Thanks. Agree with the changes.

**Pull request merged**